user-guide/Getting_started/Installing_a_DMA/Using_a_pre_installed_DataMiner_Virtual_Hard_Disk.md

```diff
@@ -127,6 +127,39 @@ Follow the below steps to configure your DataMiner Agent:
 > [!IMPORTANT]
 > For security reasons, we strongly recommend creating a second user and disabling the built-in administrator account once the setup is complete. See [Managing users](xref:Managing_users).
 
+### WSL management
```

Comment:
Where does this fit in in the procedures on this page? Is this something the user should do after the installation is complete and DataMiner is fully configured? Because right now you've added it as a subsection of "Configuring DataMiner", and that doesn't make much sense.